src/main/java/com/ratifire/devrate/util/zoom/webhook/service/ZoomWebhookService.java

```diff
@@ -36,17 +36,21 @@ public String handleZoomWebhook(String requestBody, Map<String, String> headers)
 
     String signature = headers.get("x-zm-signature");
     String timestamp = headers.get("x-zm-request-timestamp");
-
     zoomWebhookAuthService.validateSignature(signature, requestBody, timestamp);
 
     WebHookRequest payload = objectMapper.readValue(requestBody, WebHookRequest.class);
     String event = payload.getEvent();
 
-    return switch (event) {
-      case "endpoint.url_validation" -> zoomWebhookAuthService.handleUrlValidationEvent(payload);
-      case "meeting.ended" ->
+    switch (event) {
+      case "endpoint.url_validation":
+        return zoomWebhookAuthService.handleUrlValidationEvent(payload);
+      case "meeting.ended":
+        if (interviewCompletionService.validateMeetingEndTime(payload)) {
           interviewCompletionService.completeInterviewProcess(payload.getPayload().getMeeting());
-      default -> throw new ZoomWebhookException("Unknown event type");
-    };
+        }
+        return "Meeting processed successfully";
```

Comment:
Just for info. In case of successful completion of the interview process, the completeInterviewProcess method already returns the string “Interview process completed successfully!” Maybe we could use it.